src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -79,7 +79,7 @@
     private final String paymentReference;
     private final DynamicList applicantSolicitor1PbaAccounts;
     private final ClaimType claimType;
-    private final SuperClaimType superClaimType;
+    private SuperClaimType superClaimType;
```

Comment:
I can't find where superClaimType is set. Why does it has to stop being final?

Reply:
you can use CaseData.toBuilder().superClaimType(newValue).build() to get a copy of the caseData with the value you want

Reply:
that will be overwritten by the .build() at the end.

Reply:
thank you for the suggestion, I have updated the code.